src/FiniteStrain/neohook.jl

```diff
@@ -0,0 +1,37 @@
+"""
+    NeoHook(; λ, μ)
+
+Hyper elastic material, Neo-Hook
```

Comment:
Could you add a list with what the parameters mean? Like e.g. there is for `Plastic`

Reply:
Oh and maybe either put `E` and `ν` here or supply the keyword argument constructor for `λ` and `μ`. I think we shouldn't start a mess with being inconsistent between the documentation of different materials.